lib/config/configuration.go

```diff
@@ -122,7 +126,11 @@ func ApplyFileConfig(fc *FileConfig, cfg *service.Config) error {
 	if len(fc.AuthServers) > 0 {
 		cfg.AuthServers = make([]utils.NetAddr, 0, len(fc.AuthServers))
 		for _, as := range fc.AuthServers {
-			addr, err := utils.ParseAddr(as)
```

Comment:
`utils.ParseAddr` supports both `tcp://` and simple `host:port` format, no need to introduce a breaking change here